docs/topics/getting-started.md

```diff
@@ -166,7 +166,52 @@ Follow Kotlin on ![Twitter](twitter.svg){width=25}{type="joined"} [Twitter](http
 
 </tab>
 
+<tab title="Multiplatform library">
+
+Support for multiplatform programming is one of Kotlin’s key benefits. It reduces time spent writing and maintaining the same code for different platforms while retaining the flexibility and benefits of native programming.
+
+Here you'll learn how to develop and a multiplatform library:
```

Comment:
Either "and" is excess here or you wanted to add one more verb and missed it?

Reply:
Missed "publish". Fixed.